.github/workflows/opensk_build.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: OpenSK build
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build_ctap2:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: thumbv7em-none-eabi
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+      - name: Set up OpenSK
+        run: ./setup.sh
+
+      - name: Building OpenSK
```

Comment:
Is there a "cargo check" action? Otherwise I'd add the various combinations present in https://github.com/google/OpenSK/blob/master/run_desktop_tests.sh#L28-L31 as a `cargo build` here.

A Cargo check/build is also missing for the examples (https://github.com/google/OpenSK/blob/master/run_desktop_tests.sh#L34).

Reply:
Cargo check with all the variations of features is already running as a Github action: https://github.com/google/OpenSK/blob/master/.github/workflows/cargo_check.yml

Reply:
Sounds good, thanks!